setup-utils/install-mxnet-macosx.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+#set -x
+set -e
+
+if ! brew --version > /dev/null 2>&1; then
+  echo "Error: Brew is not installed or missing in your PATH"
+  exit 1
+fi
+
+function brew_install() {
+  echo "Checking for brew formula $1"
+  brew ls --versions $1 || brew install $1
+}
+# You need to have brew installed, check: http://mxnet.io/get_started/osx_setup.html
+#brew update
+brew_install cmake
```

Comment:
why not use one single brew to install multi packages to speed up the installation?